userspace/libscap/scap.h

```diff
@@ -199,6 +199,7 @@ typedef struct scap_threadinfo
 	uint64_t sid; ///< The session id of the process containing this thread.
 	char comm[SCAP_MAX_PATH_SIZE]; ///< Command name (e.g. "top")
 	char exe[SCAP_MAX_PATH_SIZE]; ///< argv[0] (e.g. "sshd: user@pts/4")
+	char exepath[SCAP_MAX_PATH_SIZE]; ///< full executable path
```

Comment:
This field doesn't seem to be serialized (and subsequently parsed) in the scap file, although it's read from `/proc` at startup. As a result, there will be inconsistencies between live and capture mode depending on where the initial state is read from. If that's what you want (which might be fine to begin with), please at least make sure that this doesn't cause spurious reads (like suggested in the other comments).